.github/workflows/cd.yml

```
@@ -19,4 +29,4 @@ jobs:
         run: make release-publish
 
       - name: Deploy release
-        run: make release-deploy
+        run: make release-deploy env_name=${{ inputs.env_name }}
```

does the parameter need to be capitalized? also nit since we're interpolating a variable, let's put quotes around it just in case it ever has stuff like spaces

```suggestion
        run: make release-deploy ENV_NAME="${{ inputs.env_name }}"
```